examples/kafka-consumer-client/kafka_consumer_client.bal

```diff
@@ -12,19 +12,14 @@ kafka:ConsumerConfiguration consumerConfiguration = {
     offsetReset: "earliest",
     // Subscribes to the topic `test-kafka-topic`.
     topics: ["test-kafka-topic"]
-
```

Comment:
Shall we keep this empty line? Otherwise in BBE page, the comment is applied to the close parentheses as well.